pkg/api/v1/meta.go

```diff
@@ -51,6 +51,9 @@ func (meta *ObjectMeta) GetDeletionGracePeriodSeconds() *int64 { return meta.Del
 func (meta *ObjectMeta) SetDeletionGracePeriodSeconds(deletionGracePeriodSeconds *int64) {
 	meta.DeletionGracePeriodSeconds = deletionGracePeriodSeconds
 }
+func (meta *ObjectMeta) GetDeletionReason() string { return "" }
```

Comment:
I think so. The `ObjectMeta` in `pkg/api/v1/meta.go` is deprecated and we add the two function here to make it satisfy the interface.